### PR TITLE
Update messages_cn.properties

### PR DESCRIPTION
**Pull request by donggyu04**
Update messages_cn.properties.

https://github.com/naver/ngrinder/issues/634#issuecomment-674619218

### CODE REVIEW

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 1]
```diff
@@ -330,7 +330,7 @@ script.message.libAndResource=\u5982\u679C\u9700\u8981\u521B\u5EFA\u5E93\u548C\u
 script.message.save.error =\u811A\u672C\u4FDD\u5B58\u5931\u8D25\u3002
 script.message.refresh.success =\u811A\u672C\u5237\u65B0\u6210\u529F\u3002
 script.message.refresh.error =\u811A\u672C\u5237\u65B0\u5931\u8D25\u3002
-script.message.empty.github.config=
+script.message.empty.github.config=script.message.empty.github.config=请您输入github配置信息。
```
Comment (songeunwoo) [inline ngrinder-controller/src/main/resources/messages_cn.properties:333]: check plz

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 2]
```diff
@@ -355,7 +355,7 @@ script.option.cookie.title=Cookies
 script.option.param.title=Params
 script.option.reqBody.title=Body
 
-script.github.add.config=
+script.github.add.config=script.github.add.config=添加github
```
Comment (songeunwoo) [inline ngrinder-controller/src/main/resources/messages_cn.properties:358]: check plz

File: ngrinder-controller/src/main/resources/messages_cn.properties (hunk 1 above)
Comment (donggyu04) [reply ngrinder-controller/src/main/resources/messages_cn.properties:333]: omg;

**Review by songeunwoo — APPROVED** [review]

**Review by junoyoon — APPROVED** [review]
Where does it come from?

**Comment by donggyu04** [comment]
> Where does it come from?

@junoyoon 

lijie1010 [helped](https://github.com/naver/ngrinder/issues/634#issuecomment-674619218) me.

**Pull request merged**